lib/test_kfence.c

```diff
@@ -162,6 +162,30 @@ static int do_test_kmalloc_aligned_oob_read(void)
 	return res;
 }
 
+static int do_test_kmalloc_aligned_oob_write(void)
+{
+	void *buffer;
+	unsigned char *c, value;
+	int res = 0;
+	const size_t size = 73;
+
+	buffer = alloc_from_kfence(size, GFP_KERNEL, SIDE_RIGHT, __func__);
+	if (buffer) {
```

Comment:
if (!buffer) return 1; 
... unindented rest...

?

Reply:
Fixed here and in the previous test.